assets/chat/js/formatters.js

```diff
@@ -20,11 +22,22 @@ class EmoteFormatter {
         if (!this.regex) {
             const emoticons = [
                 ...chat.emoticons,
-                ...chat.twitchemotes,
             ].join('|');
-            this.regex = new RegExp(`(^|\\s)(${emoticons})(?=$|\\s)`, 'gm');
+            const suffixes = 'mirror|snow|rain';
+            //const suffixes = keys(generify_options).join('|');
+            this.regex = new RegExp(`(^|\\s)(${emoticons})(:(${suffixes}))?(?=$|\s)`, 'gm');
```

Comment:
I _think_ that the second `\s` should be `\\s` like it was before.

Reply:
Playing around [here](https://regex101.com/) gives better results with one backslash but I haven't noticed any difference between them in the chat.